### PR TITLE
fix #1750

### LINKED ISSUE #1750: Tab组件想隐藏添加按钮无法实现
如题，场景如下：
image

但目前没有该配置项：
image

### PR DESCRIPTION
**Pull request by ddcat1115**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [ ] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [ ] Rebase before creating a PR to keep commit history clear.
- [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: components/tabs/index.jsx [hunk 1]
```diff
@@ -57,7 +58,11 @@ export default class Tabs extends React.Component {
       // Add new tab handler
       tabBarExtraContent = (
         <span>
-          <Icon type="plus" className={`${prefixCls}-new-tab`} onClick={this.createNewTab} />
+          {
+            !hideAdd ? (
```
Comment (afc163) [inline components/tabs/index.jsx]:
```jsx
if (!hideAdd) {
  tabBarExtraContent = ( ... )
}
```

**Comment by ddcat1115** [comment]
已修改

**Comment by afc163** [comment]
改成 `showNewTab` 如何，`showXxx` 貌似在 antd 中用的比较多。

**Comment by ddcat1115** [comment]
之前也想过，主要是考虑到，这里show是常态，hide是附加的操作，hideXXX={true}好像比showXXX={false}更直观一些

**Pull request merged**